keywords.robot

```diff
@@ -50,20 +52,6 @@ Check IPXE Appears Only Once
         FAIL    Test case marked as Failed\nRequested boot option: (${IPXE_BOOT_ENTRY}) appears not only once.
     END
 
-Launch To DTS Shell
-    [Documentation]    Launch to DTS via iPXE and open Shell.
-    Enter IPXE
-    IPXE DTS
-    Set DUT Response Timeout    120s
-    Read From Terminal Until    Enter an option
-    Set DUT Response Timeout    30s
-    Write Into Terminal    9
-    Set Prompt For Terminal    bash-5.1#
-    Read From Terminal Until Prompt
-    # These could be removed once routes priorities in DTS are resolved.
-    Sleep    10
-    Remove Extra Default Route
-
```

Comment:
IMO no point keeping this keyword - if you enter shell via pikvm like this it won't provide serial output, as explained [here](https://jira-3mdeb.atlassian.net/browse/DSH-751?focusedCommentId=129683)